eppo/src/main/java/cloud/eppo/android/EppoHttpClient.java

```diff
@@ -60,7 +52,11 @@ public void get(String path, RequestCallback callback) {
             public void onResponse(Call call, Response response) {
                 if (response.isSuccessful()) {
                     Log.d(TAG, "Fetch successful");
-                    callback.onSuccess(response.body().charStream());
+                    try {
+                        callback.onSuccess(response.body().string());
```

Comment:
Using [`string()`](https://square.github.io/okhttp/3.x/okhttp/okhttp3/ResponseBody.html#string--) to load the entire response into a string. This should be fine as long as its relatively small.